projects/jsoup/Dockerfile

```diff
@@ -15,17 +15,24 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y maven
 
-RUN git clone --depth 1 https://github.com/google/fuzzing
-RUN cat fuzzing/dictionaries/html.dict > $SRC/JsoupFuzzer.dict
-RUN cat fuzzing/dictionaries/xml.dict > $SRC/JsoupXMLFuzzer.dict
+RUN curl -L https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.zip -o maven.zip && \
```

Comment:
This installs much faster, is not affected by OS image upgrades and installs less unused dependencies.